PhysicsTools/PatAlgos/python/producersLayer1/lowPtElectronProducer_cff.py

```diff
@@ -54,10 +54,10 @@
 
 # Modifiers
 from Configuration.Eras.Modifier_fastSim_cff import fastSim
-fastSim.toModify(patLowPtElectrons, embedTrack = cms.bool(False))
+fastSim.toModify(patLowPtElectrons, embedTrack = False)
 from Configuration.Eras.Modifier_run2_miniAOD_80XLegacy_cff import run2_miniAOD_80XLegacy
-run2_miniAOD_80XLegacy.toModify(patLowPtElectrons, electronSource = cms.InputTag("gedGsfElectrons"))
-run2_miniAOD_80XLegacy.toModify(patLowPtElectrons, genParticleMatch = cms.InputTag("electronMatch"))
+run2_miniAOD_80XLegacy.toModify(patLowPtElectrons, electronSource = "gedGsfElectrons")
+run2_miniAOD_80XLegacy.toModify(patLowPtElectrons, genParticleMatch = "electronMatch")
 from Configuration.Eras.Modifier_run2_miniAOD_94XFall17_cff import run2_miniAOD_94XFall17
-run2_miniAOD_94XFall17.toModify(patLowPtElectrons, electronSource = cms.InputTag("gedGsfElectrons"))
-run2_miniAOD_94XFall17.toModify(patLowPtElectrons, genParticleMatch = cms.InputTag("electronMatch"))
+run2_miniAOD_94XFall17.toModify(patLowPtElectrons, electronSource = "gedGsfElectrons")
+run2_miniAOD_94XFall17.toModify(patLowPtElectrons, genParticleMatch = "electronMatch")
```

Comment:
this can also be shorter as 
```python
(run2_miniAOD_80XLegacy | run2_miniAOD_94XFall17).toModify(patLowPtElectrons,
     electronSource = "gedGsfElectrons",
     genParticleMatch = "electronMatch"
)
```

Reply:
Done